nilearn/image/image.py

```diff
@@ -580,6 +589,16 @@ def mean_img(
     nilearn.image.math_img : For more general operations on images.
 
     """
+    if isinstance(imgs, SurfaceImage):
+        if len(imgs.shape) < 2 or imgs.shape[1] < 2:
+            data = imgs.data
```

Comment:
Not sure what we do for Nifti images when there are no timepoints, but maybe a warning (that there are no timepoints to mean over) could be good here?

Reply:
Do we really need a warning here ? `mean_img` remains well-defined in that case.

Reply:
I'd say that if we do not send a warning in the volume case, let's not do that with surface